File: lib/src/main/java/uk/iatom/rfm69/Commands.java
package uk.iatom.rfm69;

public class Commands {
    
    public static byte writeAddr(byte address) {
        return (byte) (address | 0x7F);
    }

    public static byte readAddr(byte address) {
        return (byte) (address | 0x80);
    }
}

File: lib/src/main/java/uk/iatom/rfm69/Commands.kt
package uk.iatom.rfm69

object Commands {


    @JvmStatic fun writeAddr(address: Byte): Byte {
        return (address.toInt() or 0x7F).toByte()
    }

    @JvmStatic fun readAddr(address: Byte): Byte {
        return (address.toInt() or 0x80).toByte()
    }
}
